<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab containers="1" tab_name="tab2">
   <Container>
    <DockSplitter count="1" orientation="-" sizes="1">
     <DockArea name="...">
      <plot mode="XYPlot" style="Lines">
       <range top="1.099874" left="-1.468358" bottom="-0.375119" right="0.890659"/>
       <limitY/>
       <curve curve_y="/uav1/leader_pose_estimate/position/y" color="#1f77b4" curve_x="/uav1/leader_pose_estimate/position/x" name="/uav1/leader_pose_estimate/position/[x;y]"/>
       <curve curve_y="/uav2/leader_pose_estimate/position/y" color="#d62728" curve_x="/uav2/leader_pose_estimate/position/x" name="/uav2/leader_pose_estimate/position/[x;y]"/>
       <curve curve_y="/ugv3/leader_pose_estimate/position/y" color="#1ac938" curve_x="/ugv3/leader_pose_estimate/position/x" name="/ugv3/leader_pose_estimate/position/[x;y]"/>
       <curve curve_y="/ugv3/odom/pose/position/y" color="#ff7f0e" curve_x="/ugv3/odom/pose/position/x" name="/ugv3/odom/pose/position/[x;y]"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="MQTT Subscriber"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="true"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="ROS /rosout Visualization"/>
  <plugin status="idle" ID="ROS Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

